applications/StructuralMechanicsApplication/custom_constitutive/linear_plane_strain.cpp

```diff
@@ -98,12 +98,43 @@ void  LinearPlaneStrain::CalculateMaterialResponsePK2 (Parameters& rValues)
     }
 }
 
-//note that since we are in the hypothesis of small strains we can use the same function for everything
+//NOTE: Note that since we are in the hypothesis of small strains we can use the same function for everything
+
 void LinearPlaneStrain::CalculateMaterialResponseKirchhoff (Parameters& rValues)
 {
     CalculateMaterialResponsePK2(rValues);
 }
 
+void LinearPlaneStrain::CalculateMaterialResponsePK1 (Parameters& rValues)
+{
+    CalculateMaterialResponsePK2(rValues);
+}
+
+void LinearPlaneStrain::CalculateMaterialResponseCauchy (Parameters& rValues)
+{
+    CalculateMaterialResponsePK2(rValues);
+}
+
+void LinearPlaneStrain::FinalizeMaterialResponsePK2 (Parameters& rValues)
+{
+    // TODO: Add if necessary
```

Comment:
In that case i will not add anything, but the elements are ready for that functions (until now the elements weren't using these functions)